packages/zapp/console/src/models/Common/constants.ts

```diff
@@ -14,6 +14,7 @@ export const endpointPrefixes = {
   taskExecution: '/task_executions',
   taskExecutionChildren: '/children/task_executions',
   workflow: '/workflows',
+  dataProxyArtifactUrn: '/dataproxy/artifact_urn',
```

Comment:
just curious, what is urn states for? as looks like it used in enpoint url too

Reply:
Good question. I also had that question for Kevin 
Answer from Kevin
```
The location (s3://..) where we store deck.html or output.pb
```